tests/testthat/test-data-importer-configuration.R

```diff
@@ -1,6 +1,8 @@
 context("New DataImporterConfiguration")
 
 test_that("it can create a new data importer configuration", {
+  skip_on_os("linux") # TODO enable again as soon as npoi works under linux
+
   importerConfiguration <- DataImporterConfiguration$new()

```

Comment:
I honestly do not like the fact that so much happens on new. A constructor should be very fast and **NEVER** crash. 
I would like to see a method on that object  for importing or creating. Right now this is all implicit and it is not what I would expect